airbyte-integrations/connectors/destination-bigquery/src/test/java/io/airbyte/integrations/destination/bigquery/CdkImportTest.java

```diff
@@ -17,7 +17,7 @@ class CdkImportTest {
    */
   @Test
   void cdkVersionShouldMatch() {
-    assertEquals("0.1.4", CDKConstants.VERSION.replace("-SNAPSHOT", ""));
+    assertEquals("0.1.5", CDKConstants.VERSION.replace("-SNAPSHOT", ""));
```

Comment:
@aaronsteers can we delete this test class? looks like it was something for cdk dev work